model-engine/model_engine_server/infra/repositories/acr_docker_repository.py

```diff
@@ -36,7 +37,10 @@ def get_latest_image_tag(self, repository_name: str) -> str:
         credential = DefaultAzureCredential()
         client = ContainerRegistryClient(endpoint, credential)
 
-        image = client.list_manifest_properties(
-            repository_name, order_by="time_desc", results_per_page=1
-        ).next()
-        return image.tags[0]
+        try:
+            image = client.list_manifest_properties(
+                repository_name, order_by="time_desc", results_per_page=1
+            ).next()
+            return image.tags[0]
```

Comment:
do we want to throw an error if there are 0 image tags?

Reply:
unless that's already handled in the `ResourceNotFoundError`

Reply:
ah ok sounds good, could we note it in the code so we know why we're not gonna `IndexError`?